css/style.css

```diff
@@ -2508,6 +2514,18 @@ a.next.page-numbers:before {
   content: "\e802";
 }
 
+/* Contact page */
+
+.contact-2 .is-position-sticky {
```

Comment:
Let's not use a page slug in selector as it will move to contact (without 2) page and selector brakes.

Reply:
I was expecting that, and I have added that CSS class to the block. Would you like us to do it another way?

Reply:
it's ok with an extra class, I am just not liking contact-2. Unless there is a meaning behind a number in selector we should not do that ;)